bin/crew

```diff
@@ -557,7 +594,7 @@ def upgrade
       end
     end
 
-    if toBeUpdated.length > 0
+    if toBeUpdated.length.positive?
```

Comment:
How about `unless toBeUpdated.empty?` instead of `if toBeUpdated.length.positive?`?

Reply:
done